StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1648InheritDocMustBeUsedWithInheritingClass.cs

```diff
@@ -159,6 +159,56 @@ private static void HandleMemberDeclaration(SyntaxNodeAnalysisContext context)
                 }
             }
 
+            if (memberSyntax is ConstructorDeclarationSyntax constructorDeclarationSyntax)
+            {
+                ISymbol symbol = context.SemanticModel.GetDeclaredSymbol(constructorDeclarationSyntax, context.CancellationToken);
```

Comment:
Looks like the symbol is already available in variable "declaredSymbol" from 10+ lines above, so I don't think you need to get it again